pom.xml

```diff
@@ -63,12 +63,12 @@
       <dependency>
         <groupId>org.folio.okapi</groupId>
         <artifactId>okapi-common</artifactId>
-        <version>4.8.0</version>
+        <version>4.8.2</version>
```

Comment:
Why the increment-by-two?

Reply:
Okapi 4.8.2 is the first Okapi version that is based on Vert.x 4.1.*.

Reply:
What I meant was, why 4.8.2 rather than 4.8.1?

(I'm not saying you shouldn't do this, just interesting in what the reasoning is.)

Reply:
https://github.com/folio-org/okapi/releases/tag/v4.8.1 :
> This release identical to 4.8.0 (except version), by mistake.

https://github.com/folio-org/okapi/compare/v4.8.0...v4.8.1

Reply:
Ah, right!